tbxforms/static/sass/govuk-overrides/components/_error-summary.scss

```diff
@@ -0,0 +1,7 @@
+@import '../../govuk/core/lists';
+
+// Apply summary error styles to non-field errors too.
+.tbxforms-error-summary__list li {
```

Comment:
Is there any chance we can add a class to the `li` of non-field errors and therefore avoid nested styling here?

Reply:
Yep, [errors.html](https://github.com/torchbox/tbxforms/blob/main/tbxforms/templates/tbx/errors.html) is a file we control so we can do this.

Can you help me to understand why that's better? Adding the class to several lines within the markup, and ensuring we do that for all future changes, feels like more risk than targeting all `li`s here?

Reply:
The more you nest CSS styling rules, the harder it becomes to override them when you need to, as you either need to repeat the string of classes or use an `!important`. As a general principle we always avoid nesting styles for this reason - and BEM makes it really easy to do that. In this particular scenario it's not the end of the world, but it definitely makes for cleaner more maintainable CSS not to nest styling rules.

Reply:
Gotcha, thanks for explaining 🙌. 

Done in https://github.com/torchbox/tbxforms/pull/80/commits/36f7e66d24088f38036243c3bddbafe642754c83

Reply:
Ooh, can we avoid the double double-underscore as that's a bem anitpattern?

`tbxforms-error-summary__list__item` should be `tbxforms-error-summary__list-item`.

Once that is done, we can amend the style here to be

`tbxforms-error-summary__list-item {...}` (instead of `.tbxforms-error-summary__list li`)